spec/registry.md

```diff
@@ -144,18 +179,18 @@ Given the above description, the `:number` function is defined to work both in a
 ```
 
 Furthermore,
-`:number`'s `<matchSignature>` contains two `<match>` elements
+`:number`'s `<matchSignature>` contains multiple `<matches>` and `<when>` elements
 which allow the validation of variant keys.
-The element whose `locales` best matches the current locale
-using resource item [lookup](https://unicode.org/reports/tr35/#Lookup) from LDML is used.
-An element with no `locales` attribute is the default
-(and is considered equivalent to the `root` locale).
 
-- `<match locales="en" values="one two few other" .../>` can be used in locales like `en` and `en-GB`
-  to validate the `when other` variant by verifying that the `other` key is present
-  in the list of enumarated values: `one other`.
-- `<match ... validationRule="anyNumber"/>` can be used to valide the `when 1` variant
-  by testing the `1` key against the `anyNumber` regular expression defined in the registry file.
+- `<when option="select" values="plural"><matches><match locales="en" values="one other" ... />`
+  can be used in locales like `en` and `en-GB` if the selection type is known to be plural
+  to validate that only `one`, `other` or numeric keys are used for variants.
```

Comment:
I think we might want to make these into examples. And I think we might want to avoid "validate that only". Perhaps:

```suggestion
> For example,
> `<when option="select" values="plural"><matches><match locales="en" values="one other" ... />`
> could be used when validating translations for locales such as `en` and `en-GB`
> to check that variant keys `one` and `other` have been provided
> (in addition to any numeric keys).
```